src/Http/Client.php

```diff
@@ -442,7 +442,7 @@ protected function _sendRequest(Request $request, $options)
      * @param string $url Either a full URL or just the path.
      * @param string|array $query The query data for the URL.
      * @param array $options The config options stored with Client::config()
-     * @return string A complete url with scheme, port, host, path.
+     * @return string A complete url with scheme, port, host, and path.
```

Comment:
@davidyell Does the Queen's English use comma before "and"? :)

Reply:
https://en.oxforddictionaries.com/explore/what-is-the-oxford-comma

Reply:
Thanks for the reference.